client/src/components/payment/fee-summary.vue

```diff
@@ -94,22 +94,22 @@ export default class FeeSummary extends Vue {
 
   /* getter */
   protected get totalFees (): number {
-    return this.fees.reduce((feeTotal: number, item: any) => {
+    return this.fees instanceof Array ? this.fees.reduce((feeTotal: number, item: any) => {
```

Comment:
What is this actually doing? It looks like (lines 85-86) this.fees is initialized to an array, so when is it NOT an array?